docs/source/en/tasks/video_classification.mdx

```diff
@@ -0,0 +1,482 @@
+
+## Prepare the datasets for training
+
+For preprocessing the videos, we'll leverage the [PyTorch Video library](https://pytorchvideo.org/). We start by importing the dependencies we need. 
+
+```py 
+>>> import pytorchvideo.data
+
+>>> from pytorchvideo.transforms import (
+...     ApplyTransformToKey,
+...     Normalize,
+...     RandomShortSideScale,
+...     RemoveKey,
+...     ShortSideScale,
+...     UniformTemporalSubsample,
+... )
+
+>>> from torchvision.transforms import (
+...     Compose,
+...     Lambda,
+...     RandomCrop,
+...     RandomHorizontalFlip,
+...     Resize,
+... )
+```
+
+For the training dataset transformations, we use a combination of uniform temporal subsampling, pixel normalization, random cropping, and random horizontal flipping. For the validation and evaluation dataset transformations, we keep the transformation chain the same except for random cropping and horizontal flipping. To learn more about the details of these transformations check out the [official documentation of PyTorch Video](https://pytorchvideo.org).  
+
+We'll use the `feature_extractor` associated with the pre-trained model to obtain the following information:
+
+* Image mean and standard deviation with which the video frame pixels will be normalized.
+* Spatial resolution to which the video frames will be resized.
+
+Let's start by defining some constants.
+
+```py
+>>> mean = feature_extractor.image_mean
+>>> std = feature_extractor.image_std
+>>> resize_to = feature_extractor.size
+
+>>> num_frames_to_sample = model.config.num_frames
+>>> sample_rate = 4
+>>> fps = 30
+>>> clip_duration = num_frames_to_sample * sample_rate / fps
+```
+
+Then we can begin defining the dataset-specific transformations and the datasets respectively. Starting with the training set: 
+
+```py 
+>>> train_transform = Compose(
+...     [
+...         ApplyTransformToKey(
+...             key="video",
+...             transform=Compose(
+...                 [
+...                     UniformTemporalSubsample(num_frames_to_sample),
+...                     Lambda(lambda x: x / 255.0),
+...                     Normalize(mean, std),
+...                     RandomShortSideScale(min_size=256, max_size=320),
+...                     RandomCrop(resize_to),
+...                     RandomHorizontalFlip(p=0.5),
+...                 ]
+...             ),
+...         ),
+...     ]
+... )
+
+>>> train_dataset = pytorchvideo.data.Ucf101(
+...     data_path=os.path.join(dataset_root_path, "train"),
+...     clip_sampler=pytorchvideo.data.make_clip_sampler("random", clip_duration),
+...     decode_audio=False,
+...     transform=train_transform,
+... )
+```
+
+The same sequence of workflow can be applied to the validation and evaluation sets: 
```

Comment:
```suggestion
Apply the same workflow sequence to the validation and evaluation sets: 
```

Reply:
I meant the passive voice to denote generality. I think active voice makes it less emphasized.

Reply:
If you feel that passive voice makes it more clear in this context, feel free to leave it as is. Generally speaking, most style guides for technical documentation recommend avoiding it as much as possible.

Reply:
Yes, I am with you regarding that. It's just about one or two places (usage-wise) where I think they are useful to denote an emphasis.